neverwinter/nwscript/native/scripterrors.h

```diff
@@ -44,6 +44,7 @@
 #ifndef __SCRIPTERRORS_H__
 #define __SCRIPTERRORS_H__
 
+#define STRREF_CSCRIPTCOMPILER_ERROR_CONDITIONAL_EXPRESSION_MISSING -559
```

Comment:
This is actually a negative strref into dialog.tlk, so...
```

[558]Text:	Combine
[559]Text:	Split
[560]Text:	ERROR: UNEXPECTED CHARACTER
[561]Text:	ERROR: FATAL COMPILER ERROR
[562]Text:	ERROR: PROGRAM COMPOUND STATEMENT AT START
```

So it either needs to go into the game tlk first, or you reuse one of the existing ones... maybe `[1595]Text:	ERROR: CONDITIONAL REQUIRES SECOND EXPRESSION`?

Reply:
Probably the best answer is to remove it completely.  It was put in for the changed made to the `if-else` node trees, which are not included in this pr.  Removed in 9e1cba87.